go.spec

```
@@ -71,6 +82,11 @@ cp -a src %{buildroot}/%{_exec_prefix}
 %{_exec_prefix}/lib64/go/%{_go_rel}/bin/*
 %{_exec_prefix}/lib64/go/%{_go_rel}/pkg/*
 %{_exec_prefix}/lib64/go/%{_go_rel}/src/*
+%{_bindir}/go
+%{_bindir}/gofmt
+%{_sysconfdir}/alternatives/go
+%{_sysconfdir}/alternatives/gofmt
```

These needs to be `%ghost` files:

```suggestion
%ghost %{_sysconfdir}/alternatives/go
%ghost %{_sysconfdir}/alternatives/gofmt
```

---

Thank you! Just fixed as you suggested.

---

"Build RPM on Leap 15.4" step finished without issue. Shall we land the PR or you want to let DAOS tests finish? Thank you!